packages/e2e-tests/src/tests/invitationLink.test.ts

```diff
@@ -182,12 +182,15 @@ describe('New user joins using invitation link while having app opened', () => {
 
     it('Owner sees that guest joined community', async () => {
       console.log('Invitation Link', 21)
-      const generalChannel = new Channel(ownerApp.driver, 'general')
+      const generalChannel = new Channel(guestApp.driver, 'general')
```

Comment:
Why 'guestApp' driver? This step should check if owner sees that guest joined community. If the logic changed I'd also change test step name